CommandLine/testing/Tests/draw_test.sog/draw.edl

```diff
@@ -40,3 +40,7 @@ draw_set_color(c_white);
 // flush immediately to ensure that a 2D and 3D primitive
 // render correctly when flushed from the same batch
 draw_batch_flush();
+
+// stretched sprite should ignore origin
+draw_sprite_stretched(spr,0,0,0,25,25);

```

Comment:
There are *soooo* many sprite tests to add. Can you extract these to a new sprites test?

Reply:
Not yet, because that automatically adds about a 1-3 minute overhead. Wait until we're actually ready, I have a feeling I may need to fix some more anomalies, and this isn't hard to move at all.